_projects/2019/sunpy/helioviewer.md

```diff
@@ -13,6 +12,7 @@ issues:
 mentors:
  - wafels
  - Cadair
+ - Potentially the API creator/maintainer
```

Comment:
this won't work on the website (but then I cannot have a look as it's down for me)

Reply:
Yes it does not show up. I will remove this.